site/content/en/docs/reference/labels-and-annotations.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Labels and Annotations"
+linkTitle: "Labels and Annotations"
+date: 2024-10-09
+---
+
+This page serves as a reference for all labels and annotations in Kueue.
+
+
+### kueue.x-k8s.io/queue-name
+
+Type: Label
+
+Example: `kueue.x-k8s.io/queue-name: "my-local-queue"`
+
+Used on: Jobs.
```

Comment:
We probably shouldn’t use Jobs here…. I don’t know what we should use here. Job to me means a batch job but this label is required for any Kueue supported job (ie Job, JobSet, Ray, Pod, Deployment, etc)

Reply:
Yeah, I encountered the same question.

Reply:
Maybe "Kueue-managed Jobs."?

Reply:
This looks great.